protocols/virtualdefund/virtualdefund.go

```diff
@@ -510,6 +564,16 @@ func getSignedStatePayload(b []byte) (state.SignedState, error) {
 	return ss, nil
 }
 
+// getRequestDefundPayload takes in a serialized channel id payload and returns the deserialized channel id.
+func getRequestDefundPayload(b []byte) (types.Destination, error) {
```

Comment:
The name here should match the types at the top of the file. See my other comment about a good name (I personally prefer  `RequestFooPayload` to `FooRequestPayload`).